packages/main/src/i18n/messagebundle.properties

```diff
@@ -263,7 +263,7 @@ VALUE_STATE_ERROR_ALREADY_SELECTED=This value is already selected.
 MULTIINPUT_ROLEDESCRIPTION_TEXT=Multi Value Input
 
 #XFLD: Token number indicator which is used to show more tokens in Tokenizers inside MultiInput and MultiComboBox
```

Comment:
"which is used to show more tokens in Tokenizers inside MultiInput and MultiComboBox" not restricted to those components